BookingsApi/BookingsApi.IntegrationTests/Database/Queries/GetJusticeUsersListQueryHandlerDatabaseTests.cs

```diff
@@ -32,7 +32,17 @@ public async Task Should_return_null_when_no_users_are_found()
         [Test]
         public async Task Should_return_users_list()
         {
-            var query = new GetJusticeUserListQuery();
+            var query = new GetJusticeUserListQuery(null);
+            var users = (await _handler.Handle(query));
+
+            users.Should().NotBeNull();
```

Comment:
test description and assertion do not match

Reply:
I am not sure what you mean. However this test has not been written in this task

Reply:
The test description states a list of users should be returned but the assertion is made to check user is not null.

```suggestion
            users.Should().NotBeBeEmpty();
```

or 

```suggestion
            users.Should().BeEquivalentTo(expectedList);
```

Reply:
done